packages/adwaita_icon_theme.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Adwaita_icon_theme < Package
+  description 'Theme consisting of a set of icons for GTK+'
+  homepage 'https://gitlab.gnome.org/GNOME/adwaita-icon-theme'
+  version '3.38.0'
+  compatibility 'all'
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/adwaita-icon-theme/3.38/adwaita-icon-theme-3.38.0.tar.xz'
+  source_sha256 '6683a1aaf2430ccd9ea638dd4bfe1002bc92b412050c3dba20e480f979faaf97'
+
+  depends_on 'gtk3'
+  depends_on 'librsvg'
+  depends_on 'gdk_pixbuf' 
+  depends_on 'vala' => :build
+  
+  def self.build
+    ENV['CFLAGS'] = "-fuse-ld=lld"
+    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    ENV['GDK_PIXBUF_MODULEDIR'] = "#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders"
+    ENV['GDK_PIXBUF_MODULE_FILE'] = "#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
+    ENV['LIBRARY_PATH'] = "#{CREW_LIB_PREFIX}:/usr/$ARCH_LIB:/$ARCH_LIB:/usr/lib:/lib"
+    # Need to make sure svg support is properly loaded otherwise build fails.
+    system "LD_LIBRARY_PATH=#{CREW_LIB_PREFIX}:/usr/#{ARCH_LIB}:/#{ARCH_LIB}:/usr/lib:/lib #{CREW_PREFIX}/sbin/ldconfig"
```

Comment:
Shouldn't need to do this if the `ENV` variable is set.

Reply:
You would think not, but I've been seeing weird issues with libraries being missed that this is taking care of. Is that insane?

Reply:
Well, the `ENV` value was not set correctly in the first place so maybe that was the issue?